e2e_test/udf/wasm/src/lib.rs

```diff
@@ -1,6 +1,11 @@
 use arrow_udf::function;
 use rust_decimal::Decimal;
 
+#[function("count_char(varchar, varchar) -> int")]
+fn count_char(s: &str, c: &str) -> i32 {
```

Comment:
Is this actually used?

Reply:
No. I just added it here in case of future use.